binding-library/python/setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(
+    name='azure-functions-kafka-binding',
+    version='1.0.0',
+    packages=['kafka']
```

Comment:
One more thing actually. Tt would be best to have package like `azure_functions.kafka`. Simply having the name `kafka` could be shadowed by other packages that might also be named `kafka`.

Reply:
This should give you a folder structure like the following:

```
\---python
    |   setup.py
    |
    \---azure_functions
        \---kafka
                kafka.py
                _kafka.py
                __init__.py
```

Reply:
You could also just name it `azure_functions_kafka` if you don't want to get into the business of creating a module folder structure.

Reply:
Changed to azure_functions.kafka folder structure.
from azure_functions.kafka import KafkaEvent.